tests/app/v2/template/test_get_template.py

```diff
@@ -40,11 +40,57 @@ def test_get_template_by_id_returns_200(client, sample_service, tmp_type, expect
         'body': template.content,
         "subject": expected_subject,
         'name': expected_name,
+        'personalisation': [],
     }
 
     assert json_response == expected_response
 
 
+@pytest.mark.parametrize("create_template_args, expected_personalisation", [
+    (
+        {
+            "template_type": SMS_TYPE,
+            "content": "Hello ((placeholder)) ((conditional??yes))",
+        },
+        ["placeholder", "conditional"]
```

Comment:
do we want to identify conditionals separately in the data we return? either as a separate list, or maybe a list of dicts `[{'name': 'placeholder', 'type': 'normal'}, {'name': 'conditional', type': optional'}]`, or maybe a dict of lists `{'normal': ['placeholder'], 'optional': ['conditional']}`?